cache_images/fedora-netavark_packaging.sh

```diff
@@ -52,6 +53,7 @@ INSTALL_PACKAGES=(\
     podman
     policycoreutils
     rsync
+    rust-protoc-rust+default-devel
```

Comment:
We're not using the packaged rust or dependencies in Fedora because they tend to be very old and outdated.  See the `"Initializing upstream rust environment."`section below.  That's where you'd want to pull something like this in (assuming it's something to do with the toolchain).